Automattic/wp-calypso
# Help Center: Fix scroll to anchor

Related to https://github.com/Automattic/dotcom-forge/issues/8359

## Proposed Changes

* There were two issues
1 - The code was trying to scroll before the content was fully loaded, added a small delay to the `setTimeout` function
2 - When navigating on the same page, the scroll was getting lost because pages with anchor were not being considered.
## Why are these changes being made?


* Fixes  https://github.com/Automattic/dotcom-forge/issues/8359

## Testing Instructions



* Use live link 
* navigate to `{livelink}/hosting-features/{yousite}`
* Make sure it navigates correctly to the proper link (anchor) when the help center is opened, more details -> https://github.com/Automattic/dotcom-forge/issues/8359
* Test regressions, like Odie/Wapuu, and other internal HelpCenter results


## Pre-merge Checklist
